pages/docs/guides/validate.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Validate AsyncAPI documents"
+description: In this guide, you'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+## Introduction
+This guide teaches multiple ways to validate AsyncAPI documents. 
+
+## Create AsyncAPI document
+To validate an AsyncAPI document, you must first create one. You can create one using the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=tutorial.yml --no-tty
+```
```

Comment:
as this is guide about validation I think we do not need a section about creating asyncapi.yaml like we do in case of tutorials. I would suggest it is a `remember` note, or not event here